narwhals/_pandas_like/series.py

```diff
@@ -851,6 +851,11 @@ def rolling_mean(
     def __iter__(self: Self) -> Iterator[Any]:
         yield from self._native_series.__iter__()
 
+    def __contains__(self: Self, other: Any) -> bool:
+        if other is None:
+            return self._native_series.isna().any()  # type: ignore[no-any-return]
+        return self._native_series.isin({other}).any()  # type: ignore[no-any-return]
```

Comment:
why this over `return (self._native_series == other).any()`?

Reply:
No strong feelings 😂 I guess I went for `series.isin(...)` as I initially fell for the `other in series`

Reply:
"fell for" 😄 love that

from a quick test, it looks a lot faster
```
In [14]: %timeit s.isin({0}).any()
6.58 ms ± 687 μs per loop (mean ± std. dev. of 7 runs, 100 loops each)

In [15]: %timeit (s == 0).any()
326 μs ± 45.8 μs per loop (mean ± std. dev. of 7 runs, 1,000 loops each)
```